docker/config/executor.yaml

```diff
@@ -31,6 +31,10 @@ action:
   type: "executor"
   config: 
     executor_id: "${EXECUTOR_ID:-default}"
+    task_config:
```

Comment:
I'd rather omit this from the executor.yaml itself, since the fallback is the exact same

Reply:
Yes your right, I thought it would be more consistent to have the task_config also in the executor.yaml, because the executorId is also in there, even though the fallback for the executorId has the same value as in the config...but now I have noticed that it needs the executorId in the config because of the modifiable environment variable...😁

Thank you for updating it!